node/src/accountant/mod.rs

```diff
@@ -141,42 +167,52 @@ impl Handler<StartMessage> for Accountant {
     type Result = ();
 
     fn handle(&mut self, _msg: StartMessage, ctx: &mut Self::Context) -> Self::Result {
-        ctx.notify(ScanForPayables);
-        ctx.notify(ScanForReceivables);
+        ctx.notify(ScanForPendingPayments {});
+        ctx.notify(ScanForPayables {});
+        ctx.notify(ScanForReceivables {});
     }
 }
 
-impl Handler<ReceivedPayments> for Accountant {
+impl Handler<ScanForPayables> for Accountant {
     type Result = ();
 
-    fn handle(&mut self, msg: ReceivedPayments, _ctx: &mut Self::Context) -> Self::Result {
-        self.handle_received_payments(msg);
+    fn handle(&mut self, msg: ScanForPayables, ctx: &mut Self::Context) -> Self::Result {
+        self.scanners.payables.scan(self);
+        let _ = ctx.notify_later(msg, self.config.payables_scan_interval);
     }
 }
 
-impl Handler<SentPayments> for Accountant {
+impl Handler<ScanForPendingPayments> for Accountant {
     type Result = ();
 
-    fn handle(&mut self, msg: SentPayments, _ctx: &mut Self::Context) -> Self::Result {
-        self.handle_sent_payments(msg);
+    fn handle(&mut self, _msg: ScanForPendingPayments, ctx: &mut Self::Context) -> Self::Result {
+        self.scanners.pending_payments.scan(self);
+        self.pending_payments_notify_later_assertable(ctx);
```

Comment:
If possible I'd like to see consistency here...either this method everywhere or nowhere.